config/environments/development.rb

```diff
@@ -14,17 +14,18 @@
 
   # Enable/disable caching. By default caching is disabled.
   # Run rails dev:cache to toggle caching.
+  config.cache_store = :null_store
+  
   if Rails.root.join('tmp', 'caching-dev.txt').exist?
     config.action_controller.perform_caching = true
 
-    config.cache_store = :memory_store
```

Comment:
this should stay intact (i.e. whatever is on master), just in case we want to be debugging using dev cache at some point..